modules/virtualisation/podman/podmactl/.gitignore

```diff
@@ -0,0 +1 @@
+result
```

Comment:
No other modules need to add a .gitignore file, I don't see why this one should. You could maybe put this it into the top level gitignore file

Reply:
Reasonable suggestion.